action.yaml

```diff
@@ -57,9 +61,10 @@ inputs:
     atomic:
         description: 'Add the --atomic flag to rollback on failure'
         required: false
+
 runs:
     using: 'docker'
-    image: 'docker://bitovi/deploy-eks-helm:v1.0.3'
+    image: 'docker://bitovi/deploy-eks-helm:pr-uninstall'
```

Comment:
Make sure this is updated before the merge.

Reply:
Should I bump that to 1.0.4 or is that something you'll take care of?

Reply:
Bump to 1.0.4.  Thanks